.github/workflows/lib-build-and-push.yml

```diff
@@ -35,11 +49,12 @@ env:
  CIBW_BEFORE_TEST: "pip install -r {project}/test-requirements.txt"
  CIBW_BEFORE_BUILD_LINUX: "rm -rf ~/.pyxbld && rpm --import https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux && yum install -y libffi-devel libev libev-devel openssl openssl-devel"
  CIBW_ENVIRONMENT: "CASS_DRIVER_BUILD_CONCURRENCY=2 CASS_DRIVER_BUILD_EXTENSIONS_ARE_MUST=yes CFLAGS='-g0 -O3'"
- CIBW_SKIP: cp36* cp37* pp*i686 *musllinux*
- CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_28
- CIBW_MANYLINUX_PYPY_X86_64_IMAGE: manylinux_2_28
- CIBW_MANYLINUX_AARCH64_IMAGE: manylinux_2_28
- CIBW_MANYLINUX_PYPY_AARCH64_IMAGE: manylinux_2_28
+ CIBW_SKIP: ${{ inputs.cibw-skip }}
+ CIBW_MANYLINUX_X86_64_IMAGE: ${{ inputs.cibw-manylinux-default-image }}
```

Comment:
Why are you giving this from inputs ?

It's really overcomplicating things

Please take a look at my PR where I move all of this environment variables into pyproject.toml

No one is gonna pass those manually or otherwise

Reply:
Imo @fruch 's PR is the right direction, let's move all this to pyproject.toml.
@fruch will you be able to work on your PR?

Reply:
done, please take a look